ember-phone-input/src/components/phone-input.ts

```diff
@@ -29,10 +43,70 @@ import '../styles/styles.css';
   @public
 */
 
-export default class PhoneInputComponent extends Component {
-  @service phoneInput;
+export interface PhoneInputArgs {
+  country: string;
+  number: string | null;
+  disabled: boolean;
+  required: boolean;
+  autocomplete: string | null;
+  allowDropdown: boolean;
+  autoPlaceholder: 'aggressive' | 'off' | 'polite' | undefined;
+  customPlaceholder: string | null;
+  initialCountry: string;
+  onlyCountries: string[];
+  preferredCountries: string[];
+  separateDialCode: boolean;
+  update: (number: string | null, meta: MetaData) => void;
+  onError: (error: unknown) => void;
+}
+
+export interface PhoneInputSignature {
+  Element: HTMLInputElement;
+  Args: PhoneInputArgs;
+}
+
+export default class PhoneInputComponent extends Component<PhoneInputSignature> {
+  @service declare phoneInput: PhoneInputService;
 
+  isLoadingIti = false;
```

Comment:
If I understood correctly `Iti` means IntlTelInput, could we have the full name here? I think it's more readable.

Reply:
Yes, renamed 👍